### ISSUE TITLE
support multilingual editable text

### ISSUE THREAD

**Issue by jshorland** [ushahidi/platform#1838]
This has been a recurring issue for both enterprise and free users. They've created a hack where they input language 1 / language 2 side by side into the editable text fields (see attached image), but it often breaks our layouts (it's a game of chance depending on # of characters). 

@justin @dalezak @tuxpiper piper can you please add any context to this issue?

![screenshot 2017-06-29 09 09 16](https://user-images.githubusercontent.com/3224856/27695761-b13b11ea-5ca4-11e7-93ea-ab0811ac6936.png)

Aha! Link: https://ushahiditeam.aha.io/features/PROD-589

**Comment by justinscherer** [comment]
Some context around this:

Reading and processing text, especially on digital displays, is a relatively high-friction way to impart information (compared to an image, for example, or a common interaction affordance). 

With this in mind, having text in two languages in the same interface has several drawbacks user experience-wise:
1. It automatically increases the total information density of every page (i.e. it make things look cluttered and busy and this makes them harder to understand)
2. The cognitive burden of understanding what to do and why on any given view is higher when a user not only has to read and understand text in their own language, but also figure out which text on the page to ignore. 
3. It just looks bad and unprofessional. Imagine a major English-language brand or org had random bits of Chinese scattered throughout their interface for no apparent reason...
4. In countries like Canada, having two languages in the same interface doesn't actually a legally compliant way to support a bilingual use case. It must be either or. I assume other multi-lingual use cases are similar. 
5. It's evidence of an anglo-bias, which doesn't really jive with our international-focussed brand. 

We should fix this in .com, platform, and rollcall.

**Comment by rjmackay** [comment]
Duplicate of #1933 or related at least

**Comment by jshorland** [comment]
note: this may be upcoming depending on results of client negotiations. Popping this into backlog and creating an epic.

**Comment by rowasc** [comment]
Closing as this is related to the v5-translations work we are doing (so ends up duplicate) but tagged to be able to find the early references
